rsconnect/main.py

```diff
@@ -103,7 +111,7 @@ def failed(err):
         except EnvironmentException as exc:
             failed("Error: " + str(exc))
         except Exception as exc:
-            if click.get_current_context("verbose"):
+            if click.get_current_context(True):
```

Comment:
I believe this was an existing bug, which mypy started noticing after I made some other changes.

According to the docs, `click.get_current_context` wants a boolean, not a string. If `False`, it will throw an error if there's no context; if `True`, it'll return `None` in that situation. I set it to `True` because I think the intent here was not to throw another error.

Reply:
Let's drop the `if` condition entirely and always print the traceback in this case. It looks like the original intent was to check the `verbose` flag from the command line and only print the traceback in verbose mode. The checks for `RSConnectException` and `EnvironmentException` suppress tracebacks for known kinds of exceptions (where we provide meaningful error messages); the rest can have trackbacks for debugging.